### PR TITLE
Change progression system from number of win to xp system

### PR DESCRIPTION
**Pull request by valentinohp**

### CODE REVIEW

File: Assets/Scripts/Global/MatchHistory/MatchHistoryManager.cs [hunk 1]
```diff
@@ -63,14 +67,35 @@ private void SaveMatchHistory()
...
+        private void AddWin(int winner, Color color, int numberOfPlayer)
         {
             if (winner != -1)
             {
                 matchHistory.WinHistory[winner]++;
+                for (int i = 0; i < numberOfPlayer; i++)
+                {
+                    if (i == winner)
+                    {
+                        matchHistory.PlayerXP[i] += 100;
```
Comment (fahsaaulanh) [inline Assets/Scripts/Global/MatchHistory/MatchHistoryManager.cs:79]: kok 100?

File: Assets/Scripts/Global/MatchHistory/MatchHistoryManager.cs [hunk 2]
```diff
@@ -63,14 +67,35 @@ private void SaveMatchHistory()
...
+                    }
+                    else
+                    {
+                        matchHistory.PlayerXP[i] += 50;
```
Comment (fahsaaulanh) [inline Assets/Scripts/Global/MatchHistory/MatchHistoryManager.cs:83]: kok 50??

**Review by fahsaaulanh — APPROVED** [review]
Nicee siss

File: Assets/Scripts/Global/MatchHistory/MatchHistoryManager.cs (hunk 1 above)
Comment (valentinohp) [reply Assets/Scripts/Global/MatchHistory/MatchHistoryManager.cs:79]: silahkan baca [requirement](https://course.agate.id/mod/assign/view.php?id=461)

**Pull request merged**